misc/docerina/src/main/java/org/ballerinalang/docgen/cmd/BallerinaDocCmd.java

```diff
@@ -40,45 +37,46 @@
 /**
  * doc command for ballerina which generates documentation for Ballerina packages.
  */
-@Parameters(commandNames = "doc", commandDescription = "generate Ballerina API documentation")
+@CommandLine.Command(name = "doc", description = "generate Ballerina API documentation")
```

Comment:
generate -> generates?

Reply:
Changed all command descriptions according to https://github.com/ballerina-platform/ballerina-lang/pull/9991#discussion_r211830355